test/evergreen_develop.yml

```diff
@@ -133,3 +133,20 @@ buildvariants:
       batchtime: 40320 # 28 days
     - name: s-outdated-fixmes
       batchtime: 10080 # 7 days
+
+- name: infrequent-tests-lazyfs
+  display_name: "~ Infrequent tests with LazyFS"
```

Comment:
Just hope to point out that we don't have to create a separate build variant these days to achieve a different running frequency for given tasks. Evergreen now supports the task-level setting of test frequency (`batchtime` as used below) to override the build-variant-level setting. We could consider adding these 3 new tasks to the existing `ubuntu2004` builder. If the aim is to make LazyFS have its own builder for ease of identification please feel free to ignore my suggestion.

Reply:
Now I see there's a special cmake setting below (on line 142) to enable LazyFS, it makes sense to have a separate build variant (builder).